packages/react-dom/src/__tests__/ReactDOM-test.js

```diff
@@ -377,6 +377,26 @@ describe('ReactDOM', () => {
     }
   });
 
+  it('should not throw a TypeError when reading stateNode on ReactCurrentOwner', () => {
```

Comment:
We avoid implementation details in test titles. Just "should not crash calling findDOMNode inside a functional component".